### DISCUSSION TITLE
How does custom domain work on budibase premium (cloud) ?

### DISCUSSION THREAD

**Discussion by MikeYates** [Budibase/budibase#11734] [Help] [answered]
How does custom domain work on budibase premium (cloud) ? I am thinking about upgrading as im almost halfway with my app development, and self host is quite complicated to do.

**Comment by ConorWebb96** [comment, marked as answer]
Hey @MikeYates,

This isn't something which is available yet. The development of this has been pushed back for other development work. I don't have any set timeline for this at the moment. (When I know more I'll updated this comment.)

As for your question it wouldn't be available for Budibase premium. It would be for customers on Business and above plans.

I hope this helps.

**Reply by MikeYates** [reply]
Thanks Conor, helpful as always.
